python/ray/dag/compiled_dag_node.py

```diff
@@ -249,6 +260,8 @@ def __init__(
         self.worker_task_refs: List["ray.ObjectRef"] = []
         # Set of actors present in the DAG.
         self.actor_refs = set()
+        # record the mapping from kwarg key -> kwarg index
+        self.kwarg_to_idx = {}
```

Comment:
```suggestion
        self.kwarg_to_idx : Dict[str, int] = {}
```

Please add type hints.